internal/tui/components/chat/editor/keys.go

```diff
@@ -33,6 +38,45 @@ func DefaultEditorKeyMap() EditorKeyMap {
 			key.WithHelp("ctrl+j", "newline"),
 		),
 	}
+
+	// Override with custom keymaps if provided
```

Comment:
Instead of overriding hardcoded keymaps, wouldn’t it be better to define default keymaps with default configuration values?

Reply:
Hmm think that's how it currently works, ref: [DefaultEditorKeyMap](https://github.com/charmbracelet/crush/blob/main/internal/tui/components/chat/editor/keys.go#L14)

The overrides only kick in if the user customizes any of the commands with keymaps.

Reply:
This is just an idea, I was thinking at this exemple here : https://github.com/charmbracelet/crush/blob/025bda409311437e2c2fa174859946da2cfc1082/internal/config/config.go#L26

Reply:
yeah no I fully agree using the default keymaps makes sense, think the proposed changes uses the defaults and only overrides them if the user wants to customize their keymaps.